spec/unit/locale_spec.rb

```diff
@@ -31,6 +36,42 @@
     it "process_locale_nil" do
       expect(SgtnClient::LocaleUtil.process_locale(nil)).to eq SgtnClient::Config.configurations.default
     end
+
+
+    it "default" do
```

Comment:
Suggest to make name to be more meaningful. Usually this is a sentance to describe test cases.

Reply:
> let's change it in other PR, just keep this PR's change as small as possible.
But this code line is new. It's not chaging what is already there. Why not make it right the first time?